azurerm/data_source_key_vault.go

```diff
@@ -196,12 +196,8 @@ func dataSourceArmKeyVaultRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
-func flattenKeyVaultDataSourceSku(sku *keyvault.Sku) []interface{} {
-	result := map[string]interface{}{
-		"name": string(sku.Name),
-	}
-
-	return []interface{}{result}
+func flattenKeyVaultDataSourceSku(sku *keyvault.Sku) string {
+	return string(sku.Name)
```

Comment:
Same here,

also we don't need a function to do a simple `return string(sku.Name)`, i would have removed this entire function.

Reply:
I made it a function just in case there is more that needs to be done in the future, I will go ahead and remove the function.